components/brave_wallet_ui/common/hooks/use-multi-chain-sell-assets.ts

```diff
@@ -79,7 +77,6 @@ export const useMultiChainSellAssets = () => {
           assetSymbol: getRampAssetSymbol(sellAsset, true),
           offRampProvider: BraveWallet.OffRampProvider.kRamp,
           chainId: sellAsset.chainId,
-          address: sellAddress,
           amount: new Amount(sellAmount)
             .multiplyByDecimals(sellAsset?.decimals ?? 18)
```

Comment:
There should be no fallback to 18 as for BTC it is 8. 
Actually there should be no fallback at all as sellAsset is checked above to be truthy

Reply:
Removed the fallback!